R/rank.R

```diff
@@ -89,24 +118,39 @@ ntile <- function(x = row_number(), n) {
   }
 }
 
+#' Proportional ranking functions
+#'
+#' @description
+#' These two ranking functions implement two slightly different ways to
+#' compute a percentile. For each `x_i` in `x`:
+#'
+#' * `cume_dist(x)` counts the total number of values less than
+#'   or equal to `x_i`, and divides it by the number of observations.
```

Comment:
```suggestion
#'   or equal to `x_i`, and divides it by the number of non-missing observations.
```

Maybe something about the fact that "number of observations" doesn't include missing values here and in `percent_rank()`'s definition? Or maybe this is a separate note that goes under these bullet points?